src/pages/DataItemExplorer/index.tsx

```diff
@@ -1,11 +1,83 @@
+import { useState } from 'react';
+import { Form, Radio, Checkbox, Button, RadioChangeEvent } from 'antd';
+import dayjs from 'dayjs';
 import DataItemExplorerFolderStructure from '../../components/DataItemExplorerFolderStructure';
+import IconButton from '../../components/IconButton';
 import styles from './DataItemExplorer.module.scss';
+import DateRangePicker from '../../components/DateRangePicker';
+import { SoapRequestBoolean, SoapRequestDateType } from '../../types/api';
+import { getDefaultDateRange } from '../../utils/dateTime';
+import { Applicable, DataItemExplorerFormData } from '../../types/data';
 
 const DataItemExplorer = (): JSX.Element => {
+  const [applicable, setApplicable] = useState<Applicable>('applicableAt');
+  const [dateRange, setDateRange] = useState<[string, string]>(getDefaultDateRange());
+  const [form] = Form.useForm<DataItemExplorerFormData>();
+
+  const onFinish = ({ latestValues, applicable }: DataItemExplorerFormData): void => {
+    //TODO if gasday (applicableFor) then change fromDate time to 4am and toDate time to 3:59am
+    const fromDate = dayjs(dateRange[0]).format('YYYY-MM-DDTHH:mm:ss');
+    const toDate = dayjs(dateRange[1]).format('YYYY-MM-DDTHH:mm:ss');
+
+    const dateType =
+      applicable === 'applicableFor' ? SoapRequestDateType.gas : SoapRequestDateType.normal;
+
+    const latestFlag = latestValues ? SoapRequestBoolean.true : SoapRequestBoolean.false;
+
+    // eslint-disable-next-line no-console
+    console.log(latestFlag, fromDate, toDate, dateType);
+  };
+
+  const applicableOnChange = (values: RadioChangeEvent): void => {
+    setApplicable(values.target.value as Applicable);
+  };
+
   return (
     <div className={styles.container}>
       <span>Data Item Explorer</span>
       <DataItemExplorerFolderStructure />
+      <span>Criteria</span>
+      <Form
```

Comment:
I would say we should put this form in a separate file, just have the components in the return block in the index file?

Reply:
Keep this new refactored file in the DataItemExplorer page folder though as it's not a reusable component